CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Shit
+
+Welcome to the shit project! We're thrilled that you want to contribute to making this software less shitty.
```

Comment:
Should contributions make it less shitty, or _more_ shitty? 🤔

Reply:
@keith-kurak Yes.